daml-lf/spec/daml-lf-1.rst

```diff
@@ -2982,7 +2973,7 @@ as described by the ledger model::
      'tpl' (x : T)
          ↦ { 'choices' { …, 'choice' ChKind Ch (y : 'ContractId' Mod:T) (z : τ) : σ 'by' eₚ 'observers' eₒ ↦ eₐ, … }, … }  ∈  〚Ξ〛Mod
      cid ∈ dom(st₀)
-     st₀(cid) = (Mod:T, vₜ, 'inactive')
+     st₀(cid) = (Mod':T', vₜ, 'inactive')
```

Comment:
As mentioned in person, I am not sure if we really evaluate controllers before checking for template id mismatches.

Reply:
I  think you are right. It should be just after Inactive rule.